test/Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.FunctionalTests/DatabaseErrorPageMiddlewareTest.cs

```diff
@@ -77,8 +77,9 @@ public virtual Task Invoke(HttpContext context)
             }
         }
 
-        [ConditionalTheory]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [ConditionalFact(Skip = "aspnet/Diagnostics#350")]
+        [OSSkipCondition(OperatingSystems.Linux)]
+        [OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
@bricelam until the recent build infrastructure changes, these tests weren't attempted. There are 18 tests in this assembly but
```

info:   Finished:    Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.FunctionalTests
info: === TEST EXECUTION SUMMARY ===
info:    Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.FunctionalTests  Total: 6, Errors: 0, Failed: 0, Skipped: 0, Time: 0.544s
```

But, do you have a better alternative for tests using LocalDB?